drivers/adc/adc_stm32.c

```diff
@@ -1620,6 +1676,8 @@ static const struct adc_driver_api api_stm32_driver_api = {
 		(LL_ADC_CLOCK_SYNC_PCLK_DIV),		\
 		(LL_ADC_CLOCK_ASYNC_DIV))
 
+#define ADC_STM32_CLOCK_IS_SYNC(x) COND_CODE_1(IS_EQ(ADC_STM32_CLOCK(x), SYNC), (true), (false))
```

Comment:
Please reuse existing macro definition style to ease things for reader:
```suggestion
/* Help text */
#define ADC_STM32_CLOCK_IS_SYNC(x)			\
	COND_CODE_1(IS_EQ(ADC_STM32_CLOCK(x), SYNC), 	\
		(true), 				\
		(false))
```

Reply:
I have removed this and essentially determine the sync or async clock based on the clk_prescaler setting during runtime.